config/jobs/periodic/kubeflow/kubeflow-pipelines-component.yaml

```diff
@@ -0,0 +1,23 @@
+  periodics:
+  - name: kubeflow-pipelines-component-yaml-ppc64le
+    cluster: kubeflow-ppc64le-cluster
+    labels:
+      preset-kubeflow-powervs: "true"
+    decorate: true
+    decoration_config:
+      gcs_configuration:
+        bucket: kubeflow-upstreamci-bucket
+        path_strategy: explicit
+      gcs_credentials_secret: kubeflow-ppc64le-s3-credentials
+    cron: "0 12 * * *"
+    extra_refs:
+      - base_ref: master
+        org: valen-mascarenhas14
```

Comment:
Why not use the upstream repo https://github.com/kubeflow/pipelines/.
I see your fork https://github.com/valen-mascarenhas14/pipelines is not ahead by any commit.

Reply:
I've added the upstream repo now